appyx-components/internal/common/build.gradle.kts

```diff
@@ -33,7 +37,7 @@ kotlin {
         }
         val androidMain by getting
         val desktopMain by getting
-        val jsMain by getting
+//        val jsMain by getting
```

Comment:
any reason behind this, or is it just an accident?

Reply:
Good catch. It's not harmful but I'll uncomment it.